src/components/Books.jsx

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable react/prop-types */
```

Comment:
- [ ] @AJ193 it's not good practice disabling linter check in your file, kindly consider try solving the problem causing the linter error instead of disabling the linter check.

`hint`

``an example code on how to properly use props``
```suggestion
import PropTypes from 'prop-types';

const Book = ({ title, author }) => (
  <div >
      `your code using the props`
  </div>
);

Book.propTypes = {
  title: PropTypes.string.isRequired,
  author: PropTypes.string.isRequired,
};

export default Book;
```